lib/Order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu.rb'
+
+# This new class is going to create an order
+class Order
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = []
+    @menu = Menu.new
```

Comment:
Good that an instance of menu class is initialised here, keeps code DRY, stops you having to initialise every time you use Menu within this Order class